variables.tf

```diff
@@ -0,0 +1,83 @@
+variable "region"     {
+  description = "AWS region"
+  default     = "eu-west-1"
+}
+
+variable "zones" {
+  description = "AWS availability zones"
+  default     = {
+    zone0 = "eu-west-1a"
+    zone1 = "eu-west-1b"
+    zone2 = "eu-west-1c"
+  }
+}
+
+variable "vpc_cidr" {
+  description = "CIDR for VPC"
+  default     = "10.0.0.0/16"
+}
+
+variable "infra_cidrs" {
+  description = "CIDR for infrastructure subnet indexed by AZ"
+  default     = {
+    zone0 = "10.0.0.0/24"
+    zone1 = "10.0.1.0/24"
+    zone2 = "10.0.2.0/24"
+  }
+}
+
+variable "ubuntu_amis" {
+  description = "Base AMI to launch the instances with"
+  default = {
+    eu-west-1 = "ami-234ecc54"
+    eu-central-1 = "ami-9a380b87"
+  }
+}
+
+
+variable "health_check_interval" {
+  description = "Interval between requests for load balancer health checks"
+  default     = 5
+}
+
+variable "health_check_timeout" {
+  description = "Timeout of requests for load balancer health checks"
+  default     = 2
+}
+
+variable "health_check_healthy" {
+  description = "Threshold to consider load balancer healthy"
+  default     = 2
+}
+
+variable "health_check_unhealthy" {
+  description = "Threshold to consider load balancer unhealthy"
+  default     = 2
+}
+
+variable "elb_idle_timeout" {
+  description = "Timeout idle connections after 300 seconds"
+  default     = 300
+}
+
+variable "dns_zone_id" {
+  description = "Amazon Route53 DNS zone identifier"
+  default = "Z3SI0PSH6KKVH4"
+}
+
+variable "dns_zone_name" {
+  description = "Amazon Route53 DNS zone name"
+  default     = "paas.alphagov.co.uk"
```

Comment:
Speaking of dev/staging/prod, we might want to use different zones for these. As such, we could say that the current `cf` subdomain is for dev. Or go ahead and create AWS accounts, zones and setup delegation for dev/staging/prod already. I think we'd be specifying the zone by parameter then or have some logic to pick zone automatically based on env. name.